tests/Stubs/patch/https---api-v1-shipments-new-shipment.json

```diff
@@ -0,0 +1,9 @@
+{
+  "data": {
+    "type": "shipments",
+    "id": "new-shipment",
```

Comment:
I don't think you should be able to update the id on a shipment right?

Reply:
It's not possible to update an id, but you still need to provide a type and id when patching a shipment (even though you also have to add it to the path).